submissions/ArthurGorbenko/HTML5-Form/index.html

```diff
@@ -0,0 +1,487 @@
+          <div class="form-wrapper_radio_tone_urgent">
+            <section class="form-select_tone">
+              <span class="form-sections_headers_text form-select_tone_header"
+                >Select your tone</span
+              >
+              <input
+                type="radio"
+                name="tone"
+                id="casual"
+                class="visually__hidden"
+              />
+              <label for="casual" class="button form-select_tone_radio_label">
+                <span class="form-select_tone_icon casual"
+                  >Friendly/casual</span
+                >
+              </label>
+              <input
+                type="radio"
+                name="tone"
+                id="corporate"
+                class="visually__hidden"
+              />
+              <label
+                for="corporate"
+                class="button form-select_tone_radio_label"
+              >
+                <span class="form-select_tone_icon pro">Corporate/pro</span>
+              </label>
+              <input
+                type="radio"
+                name="tone"
+                id="formal"
+                class="visually__hidden"
+              />
+              <label for="formal" class="button form-select_tone_radio_label">
+                <span class="form-select_tone_icon formal">Media/formal</span>
+              </label>
+            </section>
+            <section class="form-range_urgent">
+              <span
+                class="form-sections_headers_text form-sections_headers_text_urgent"
+              >
+                Tell us how urgent this job is
+              </span>
+              <div class="wrapper_form-range">
+                <input
+                  type="range"
+                  name="urgent"
+                  id=""
+                  min="0"
+                  max="2"
+                  step="1"
+                  value="1"
+                  class="form-range_input"
+                />
+                <div class="range_text">
+                  <span>I got time</span>
+                  <span class="orange">Average</span>
+                  <span>Yesterday</span>
+                </div>
+              </div>
+              <aside class="form-urgent_info">
+                <span class="form-urgent_info_header">Timeline info</span>
+                <p class="form-urgent_info_text">
+                  Lorem ipsum dolor sit <span class="orange">Orange</span>.Lorem
+                  ipsum dolor sit amet.Lorem ipsum dolor sit amet.
+                </p>
+              </aside>
+            </section>
+          </div>
+          <section class="form-sign_pay_method">
+            <span
+              class="form-sections_headers_text form-sign_pay_method_header"
+            >
+              Sign up and choose a payment method
+            </span>
+
+            <span class="form-sign_pay_method_link"
+              >Already have an account with us ?<a
```

Comment:
Fix alignment of this section. [Image](http://prntscr.com/oczk85)

Reply:
Done.